MuseumKulturenBasel/MusKulB26282.xml

```diff
@@ -29,12 +29,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         <summary/>
                         <msItem xml:id="ms_i1">
                             <title type="complete" />
-                            <textLang mainLang="gez">Protective prayer with ʾasmāt</textLang>
+                            <textLang mainLang="gez" corresp="LIT5888AsmatPrayer">Protective prayer with ʾasmāt</textLang>
```

Comment:
this should be `title `and not `textLang`

Reply:
Oh, sorry!
Corrected